lib/auth0/mixins/initializer.rb

```diff
@@ -30,9 +30,9 @@ def initialize_api(options)
       end
 
       def base_url(options)
-        domain = options[:domain] || options[:namespace]
-        fail InvalidApiNamespace, 'Api namespace must supply an API domain' if domain.nil?
-        "https://#{domain}"
+        @domain = options[:domain] || options[:namespace]
```

Comment:
We need @domain here, because we are creating the authorization and logouts urls in the authentication_endpoints. The same happens with @client_id.  We followed the same pattern.